contrib/mesos/pkg/executor/executor.go

```diff
@@ -179,6 +166,10 @@ func New(config Config) *KubernetesExecutor {
 	return k
 }
 
+func (k *KubernetesExecutor) InitialRegComplete() <-chan struct{} {
```

Comment:
I prefer RegistrationComplete. Registration is not intuitive. The channel is not a registration.